examples/todo/Main.hs

```diff
@@ -142,6 +144,12 @@ buildUI wenv model = widgetTree where
         filler
       ] `styleBasic` [bgColor (grayDark & L.a .~ 0.5)]
 
+  confirmDeleteLayer = case model ^.action of
```

Comment:
The todo example now includes a confirm popup when deleting a todo, because the confirm alert was not previously in the examples anywhere (and I wanted to test because I added the shadow to it).